### PR TITLE
iterate over points and close each object in the loop one by one

### PR DESCRIPTION
**Pull request by kmarcisz**
Fixes https://github.com/inventables/dxfimport/issues/8
Last attempt to fix the issue did not iterate over points in the path but worked on the top level array which does not contain the actual points, but is a series of arrays with points, one for each top level shape.

This PR fixes the problem.

Tested by importing the files from #8 and seeing it import without issue, with the right cut type, and being able to change cut types.
Added a test for the function using a hack which requires small change to the files prior to deploying a new version - suggestions for refactoring welcomed. Also this is not hooked up to any CI yet, we might want to add a simple CircleCI integration.

### CODE REVIEW

**Comment by kmarcisz** [comment]
I've set up github workflow to test the code. I had to downgrade the mocha version as the latest one is not compatible with node8. Once we move over to node10 or 12 we should bump the mocha version at the same time.

Not sure if node@14 test is necessary.

**Comment by kmarcisz** [comment]
I have carved a piece after using the new syntax and the result was accurate. I had selected my desired cut specifications which was preserved from 2d to 3d to carved piece.

File: easel/app.js [hunk 1]
```diff
@@ -74,10 +78,13 @@ function sortPoints(points) {
   }
 
   if (points.length) {
-    // close the path if first and last point are close enough to make the path open due to dxf -> svg floating point inaccuracies
-    if (Math.abs(points[0].x - points[points.length - 1].x) < diff && Math.abs(points[0].y - points[points.length - 1].y) < diff ) {
-      points[points.length - 1] = points[0];
-    }
+    points = points.map(subPoints => {
+      // close the path if first and last point are close enough to make the path open due to dxf -> svg floating point inaccuracies
+      if (Math.abs(subPoints[0].x - subPoints[subPoints.length - 1].x) < diff && Math.abs(subPoints[0].y - subPoints[subPoints.length - 1].y) < diff ) {
```
Comment (hphu3) [inline easel/app.js:83]: Did the structure of points change? I'm not seeing any changes to it from the diff but from the old code it looks like points is an array of subpoints `[{ x, y }]` but from the changes it's being accessed like an array of subpoint arrays `[[{x, y }, {x, y}], [{x, y}]]`

Comment (kmarcisz) [reply easel/app.js:83]: The old code in https://github.com/inventables/dxfimport/pull/7 was wrong, and the old changes did not fix the issue.
If you look at https://github.com/inventables/dxfimport/blob/e118e3686d1cab89787eede7302f07f4194641a4/easel/app.js#L50 you can see the structure has always been an array of subpoints. Do not use https://github.com/inventables/dxfimport/pull/7 as a reference please.

**Pull request merged**